invenio_communities/templates/semantic-ui/invenio_communities/details/subcommunity-invitation/index.html

```diff
@@ -0,0 +1,47 @@
+{# -*- coding: utf-8 -*-
```

Comment:
Comment: Would be nice to see a screenshot of this 😊

Reply:
Agree, though actually this file is not used at all when looking at the similar usage for `../../subcommunities/index.html`, so I think it can be removed.